docs/playground.md

```diff
@@ -0,0 +1,252 @@
+---
+slug: playground
+append_help_link: true
+title: Semgrep Playground
+description: "Semgrep Playground is a beginner-friendly environment for creating your first rules."
+hide_title: true
+
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Creating rules with Semgrep Playground
+
+
+Semgrep Playground is a live editor used to create and test rule patterns on sample code. By testing the pattern out on sample code, you are able to quickly assess the purpose, utility, and speed of a rule as well as save it for later refinement, reuse, or for sharing.
+
+
+## Creating a rule
+
+There are two ways to create a rule:
+
+* **Simple view.** The simple view is best for quick and simple pattern-matching use cases, but does not display all of Semgrep's operators.
+* **Advanced view.** The advanced view provides the minimum required keys for a Semgrep rule and expects users to fill in additional keys, such as pattern operators or metadata, to complete the rule.
+
+
+### Using simple view

```

Comment:
When I think of the main user story for the simple view, it's to explore Semgrep's language (through its pattern operators and language fields) because the simple view shows the possible values through a drop down box. 

How about:
"Exploring pattern operators and syntax through simple view"
"Learning Semgrep basics through simple view" might be more succinct?

Reply:
@s-santillan I like "Learning Semgrep basics through simple view". You can even use: "Understanding Semgrep basics through simple view". :-) But if the sole purpose is mostly for the easy start with Semgrep rules, feel free to use learning. :-)

Reply:
Added in latest update :)